### ISSUE TITLE
Don't report an error if .vscode/settings.json doesn't exist

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#303]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 0.4.0
-   ESLint version: N/A
-   Node version: N/A

#### Actual Behavior

```
λ npx tslint-to-eslint-config
❌ 1 error running tslint-to-eslint: ❌
Error: ENOENT: no such file or directory, open 'path\to\dir\.vscode\settings.json'
```

#### Expected Behavior

If the project doesn't have the file, don't complain.

#### Reproduction

In a new empty directory:

```
npx eslint --init
npx tslint-to-eslint-config
```
